app/views/users/mailer/confirmation_instructions.html.erb

```diff
@@ -1,5 +1,6 @@
-<p>Welcome <%= @email %>!</p>
+<p><%= t('.greeting', :recipient => @resource.email, :default => "Welcome #{@resource.email}!") %></p>
 
-<p>You can confirm your account email through the link below:</p>
+<p><%= t('.instruction', :default => "You can confirm your account email through the link below:") %></p>
 
-<p><%= link_to 'Confirm my account', confirmation_url(@resource, confirmation_token: @token) %></p>
+<p><%= link_to t('.action', :default => "Confirm my account"),
+         confirmation_url(@resource, :confirmation_token => @resource.confirmation_token) %></p>
```

Comment:
missing newline

We really need a linter for `.html.erb` :smile:

Reply:
```suggestion
         confirmation_url(@resource, :confirmation_token => @resource.confirmation_token) %></p>

```